src/components/api.jsx

```diff
@@ -0,0 +1,17 @@
+import axios from "axios";
+
+const ncnewsApi = axios.create({
+  baseURL: "https://be-nc-news-whiy.onrender.com/api",
+});
+
+export const getArticles = () => {
+  return ncnewsApi.get("/articles").then(({ data }) => {
+    return data.articles;
+  });
+};
+
+export const getArticle = (article_id) => {
+  return ncnewsApi.get(`/articles/${article_id}`).then(({ data }) => {
+    return data.article;
+  });
+};
```

Comment:
Nicely done 👍,  Just as a heads up this file should not end with the .jsx file extension as it is  not a react component, but instead can be a regular js file ending with .js.